apps/avifenc.c

```diff
@@ -1016,9 +1014,7 @@ int main(int argc, char * argv[])
             returnCode = 1;
             goto cleanup;
         }
-    }
 
-    if (!gridDimsCount) {
```

Comment:
This if block can be merged with the else block above because they have the same condition.